lib/components/artist/shows/metadata.js

```diff
@@ -55,8 +54,8 @@ export default Relay.createContainer(Metadata, {
       fragment on PartnerShow {
         kind
         name
-        start_at
-        end_at
+        exhibition_period
+        status
```

Comment:
Ah, I have been working on this! It took me quite some time to find it, but now I think I know what needs to be done.